db/warehouse/migrate/20230830121811_nullable_ce_hmis_participation_cols.rb

```diff
@@ -0,0 +1,13 @@
+class NullableCeHmisParticipationCols < ActiveRecord::Migration[6.1]
+  def change
+    # change_column_null :HMISParticipation, :HMISParticipationType, true
+    change_column_null :HMISParticipation, :HMISParticipationStatusStartDate, true
+    change_column_null :HMISParticipation, :UserID, true
+    change_column_null :HMISParticipation, :ExportID, true
+
+    # change_column_null :CEParticipation, :AccessPoint, true
+    # change_column_null :CEParticipation, :CEParticipationStatusStartDate, true
```

Comment:
These commented-out ones are not strictly necessary to change, I guess, but might be a good idea given that other tables are mostly nullable (they do not match the HUD spec w.r.t. nullability).